src/input/InputEventMapper.cc

```diff
@@ -140,13 +144,29 @@ void InputEventMapper::onTimer()
     if ((fabs(rxVPRel) > threshold) || (fabs(ryVPRel) > threshold) || (fabs(rzVPRel) > threshold)) {
         InputEvent *inputEvent = new InputEventRotate2(rxVPRel, ryVPRel, rzVPRel);
         InputDriverManager::instance()->postEvent(inputEvent);
+        any = true;
     }
     
     double z = (getAxisValue(zoom)+getAxisValue(zoom2))*zoomGain;
     if (fabs(z) > threshold) {
         InputEvent *inputEvent = new InputEventZoom(z);
         InputDriverManager::instance()->postEvent(inputEvent);
+        any = true;
+    }
+
+    return any;
+}
+
+void InputEventMapper::considerGeneratingDeferredEvents()
+{
+    if (!timer->isActive()) {
+        timer->start(30);
```

Comment:
It does not work for me (Debian/testing, Qt5.12.5), just getting
```
QObject::startTimer: Timers cannot be started from another thread
```
This still reports at least the buttons to the 3d view, but the preferences dialog does not get any events in the Axis and Buttons tabs.

Using this to forward the start call to the thread the timer is running on makes it work for me:
```suggestion
        QMetaObject::invokeMethod(timer, "start", Qt::QueuedConnection, Q_ARG(int, 30));
```